src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/PlatformLogger.java

```diff
@@ -0,0 +1,18 @@
+package com.microsoft.maui;
+
+import android.util.Log;
+
+public class PlatformLogger {
+
+    private final String tag;
+    public final boolean isVerboseLoggable;
+    
+    public PlatformLogger(String tag) {
+        this.tag = tag;
+        this.isVerboseLoggable = Log.isLoggable(tag, Log.VERBOSE);
```

Comment:
Your consideration is correct, but I see no value in executing `Log.isLoggable(.., ...)` every time we need to log something, especially because this logging is only useful to debug issues.
Considering end users will not enable logging and issues must be reproducible I think it's wise to reduce as much as possible the cost of logging while not needed.
This is why the configuration is being read only once at startup.
This is the same approach used by `Glide`.